File: sdkwork-app-sdk-csharp/Models/ServiceStatus.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class ServiceStatus
    {
        public string? Api { get; set; }
        public string? Database { get; set; }
        public string? Cache { get; set; }
        public string? Storage { get; set; }
        public string? ThirdParty { get; set; }
    }
}

File: sdkwork-app-sdk-typescript/src/types/service-status.ts
/** 服务状态 */
export interface ServiceStatus {
  /** API服务状态 */
  api?: string;
  /** 数据库服务状态 */
  database?: string;
  /** 缓存服务状态 */
  cache?: string;
  /** 存储服务状态 */
  storage?: string;
  /** 第三方服务状态 */
  thirdParty?: string;
}
